### ISSUE TITLE
Clipboard commands in the integrated terminal of VS Code

### ISSUE THREAD

**Issue by jfromaniello** [rbreaves/kinto#285]
**Describe the bug**

**Expected behavior**
Clipboard commands in the integrated terminal for *VS Code* don't work. I think Super+C/V should work as they do on the gnome terminal. I tried some changes in the kinto.py file but I am not familiar with this yet and I wasn't able to get it working.

**Install Type:** Bare Metal
**Distro:** Ubuntu 20.04.1 LTS
**DE:** Gnome
**Branch:** master
**Commit:** 1bdef2f

Thank you in advance! I really like this project.

**Comment by rbreaves** [comment]
That'll be a difficult thing to implement a fix for unfortunately because when the cursor ends up inside the terminal portion of VS Code there isn't anything that I am aware of that I can use to easily trigger the proper mapping. Xkeysnail still believes you are simply in VS Code due tot he wmname property not changing.

I've explored this issue awhile back but never really found a solution for it. I did implement the actual Ctrl-C combo to occur still in the editor as a whole so that sigints will still work.

Closing this ticket as it is on my radar and mentioned in a roadmap ticket already.
